.github/workflows/docker.yml

```diff
@@ -0,0 +1,47 @@
+name: Build Docker images 
+
+on:
+  push: 
+    branches:
+      - master
+    tags:
+      - '*'
+
+jobs: 
+  build:
+    name: Build Docker image 
+    runs-on: ubuntu-latest
+    environment: scipy-dev
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+      - name: Lint Docker 
+        uses: brpaz/hadolint-action@v1.2.1
+        with: 
+          dockerfile: ./tools/Docker-dev/Dockerfile
+      - name: Get refs
+        shell: bash
+        run: |
+          export raw_branch=${GITHUB_REF#refs/heads/}
+          echo "::set-output name=branch::${raw_branch//\//-}"
+          echo "::set-output name=date::$(date +'%Y%m%d')"
+          echo "::set-output name=sha8::$(echo ${GITHUB_SHA} | cut -c1-8)"
+        id: getrefs
+      - name: Set up QEMU
```

Comment:
Why is QEMU needed? It tends to be slow and not completely reliable, native builds should work just fine here right?

Reply:
Ah yes - we could use just native builds (carried this over from other action I have in another repo, will remove it)